clients/js/README.md

```diff
@@ -825,6 +837,8 @@ Commands:
   worm sui upgrade                          Generate contract upgrade VAA
   worm sui attestation                      Generate a token attestation VAA
   worm sui recover-chain-id                 Generate a recover chain ID VAA
+  worm sui set-default-delivery-provider    Sets the default delivery provider
```

Comment:
ideally this wouldn't be here, but don't think I have a choice given how this is generated :(